### PR TITLE
Check pep8 syntax before consuming CPU time

### PR DESCRIPTION
**Pull request by Zempashi**

### CODE REVIEW

File: tests/utils/kube.py [hunk 1]
```diff
@@ -29,24 +29,24 @@ def _find_resource_name(resource):
 
 def get_kube_resources(kubeconfig, resource, namespace='default', name=None):
     k8s_config.load_kube_config(config_file=kubeconfig)
-    resource_name, api_name, namespaced_resource = _find_resource_name(resource)
+    resrc_name, api_name, namespaced_resource = _find_resource_name(resource)
```
Comment (NicolasT) [inline tests/utils/kube.py]: A somewhat more 'standard' shorthand for `resource` is `res` rather than `resrc`.

File: eve/main.yml [hunk 2]
```diff
@@ -34,6 +35,11 @@ stages:
         method: clobber
         retryFetch: true
         haltOnFailure: true
+    - ShellCommand:
+        name: 'Check syntax'
+        haltOnFailure: true
+        command: |-
+          tox -e pep8
```
Comment (NicolasT) [inline eve/main.yml]: Could also be a Docker container with all deps pre-installed, even using Alpine.

Comment (Zempashi) [reply eve/main.yml]: Yes was doing that ;)

File: eve/tests/single-node/init.sh [hunk 3]
```diff
@@ -13,7 +13,3 @@ if systemctl is-enabled --quiet iptables; then
 fi
 iptables -F || die
 iptables -X || die
-
```
Comment (Zempashi) [inline eve/tests/single-node/init.sh:16]: With kubernetes worker this is not strictly needed anymore. But I find it better like that, so I let that part

**Comment by Zempashi** [comment]
@bert-e reset

**Comment by Zempashi** [comment]
Here, would like a tacit approval for this PR. The build on 1.1 is failling because of other pep8 issue (specific in 1.1), but If need to recreate the integration branch, I will have to re-apply the fix and change the branch each time. Would like to fix that only at the end.

File: eve/main.yml [hunk 4]
```diff
@@ -69,3 +70,18 @@ stages:
...
+      path: eve/workers/pep8.yaml
+      images:
+        pep8-checker:
+          context: '.'
+          dockerfile: eve/images/pep8-checker/Dockerfile
+    steps:
+    - Git: *git_pull
+    - ShellCommand:
+        name: 'Check syntax'
+        haltOnFailure: true
```
Comment (NicolasT) [inline eve/main.yml:86]: Would this be required on the doc stage as well?

File: eve/workers/pep8.yaml [hunk 5]
```diff
@@ -1,17 +1,23 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: "zenko-test-pod"
+  name: "metal-k8s-pep8"
```
Comment (NicolasT) [inline eve/workers/pep8.yaml]: `metalk8s`

**Review by NicolasT — DISMISSED** [review]

File: eve/main.yml (hunk 4 above)
Comment (Zempashi) [reply eve/main.yml:86]: May be. I'll open a PR for this change

**Comment by Zempashi** [comment]
@bert-e reset

**Comment by Zempashi** [comment]
@bert-e reset

**Comment by Zempashi** [comment]
@bert-e force_reset

**Review by NicolasT — DISMISSED** [review]

**Comment by Zempashi** [comment]
@bert-e wait

**Comment by Zempashi** [comment]
@bert-e force_reset

**Comment by NicolasT** [comment]
Does this now include #360?

**Review by NicolasT — APPROVED** [review]
:crossed_fingers:

**Pull request merged**